zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js

```diff
@@ -17,7 +17,7 @@
 
 angular.module('zeppelinWebApp')
   .controller('ParagraphCtrl', function($scope,$rootScope, $route, $window, $element, $routeParams, $location,
-                                         $timeout, $compile, websocketMsgSrv) {
+                                         $timeout, $compile, websocketMsgSrv, Notification) {
```

Comment:
thanks! I can't find it under zeppelin-web\bower_components\angular-ui-notification folder. am I missing something?